docs/source/standard/contract.rst

```diff
@@ -94,3 +94,111 @@ Schema
 
     |ocdsDescription|
     All documents and attachments related to the contract, including any notices.
+
+.. _ContractTerms:
+
+ContractTerms
+=============
+
+Schema
+------
+
+:contractType:
+    string, required
+
+    The only choice currently is ``lease``.
+
+:leaseTerms:
+    :ref:`LeaseTerms`
+
+    The options of property lease.
+
+.. _LeaseTerms:
+
+LeaseTerms
+==========
+
+Schema
+------
+
+:leaseDuration:
+    ISO8601 duration object, required
+
+    Property lease duration in contract.
+
+:taxHolidays:
+    list of :ref:`TaxHolidays`
+
+    Special rated period of lease.
+
+:escalationClauses:
+    list of :ref:`EscalationClauses`
+
+.. _TaxHolidays:
+
+TaxHolidays
+===========
+
+Schema
+------
+
+:taxHolidaysDuration:
+    ISO8601 duration object, required
+
+    Duration of rent holidays.
+
+:conditions:
+    string, required
+
+    Lease rent holidays conditions in Ukrainian.
+
+:conditions_en:
+    string
+
+    Lease rent holidays conditions in English.
+
+:conditions_ru:
+    string
+
+    Lease rent holidays conditions in Russian.
+
+:value:
+    :ref:`Value` object, required
+
+    The lease value for tax holidays.
+
+.. _EscalationClauses:
+
+EscalationClauses
+=================
+
+Schema
+------
+
+:id:
+    string, auto-generated, read-only
+
+:escalationPeriodicity:
+    Time intervals, required
+
+    Periodicity of escalation.
```

Comment:
Periodicity within which the payment will be increasing in accordance to inflation rate (for example) on a yearly basis.